internal/errs/errs.go

```diff
@@ -36,6 +36,33 @@ type TransientError interface {
 	IsTransient()
 }
 
+type UserFacingError interface {
+	error
+	UserError() string
+}
+
+type userFacingError struct {
+	wrapped error
+	message string
+	tips    []string
+}
+
+func (e *userFacingError) Error() string {
+	return JoinMessage(Wrap(e.wrapped, e.message))
```

Comment:
This leads to what feels like an awkward UX with the "User Facing Error: " string before the actual message. Do we want to keep this or remove it?

```
❯ ~/work/cli/build/state _hello
█ Saying hello (Beta)

Beta Feature: This feature is still in beta and may be unstable.

This command is for example use only
█ Something Went Wrong

 x User Facing Error: Cannot say hello because you are not in a project directory

█ Need More Help?

 • Try using `state checkout` first
 • Ask For Help → https://community.activestate.com/c/state-tool/
```